File: LeagueOfNews.Core/Model/Newsfeed.cs
using LeagueOfNews.Core.Interface;
using PropertyChanged;

namespace LeagueOfNews.Model
{
    [AddINotifyPropertyChangedInterface]
    public class Newsfeed
    {
#nullable enable
        public string? UrlToNewsfeed { get; set; }
        public string? Date { get; set; }
        public string? Title { get; set; }
        public string? ShortDescription { get; set; }
        public byte[]? Image { get; set; }
        public string? ImageUri { get; set; }
#nullable disable
        public NewsCategory Page { get; set; }
        public NewsWebsite Website { get; set; }
    }
}

File: league_of_news/lib/features/newsfeeds/domain/entities/newsfeed.dart
import 'package:meta/meta.dart';

class Newsfeed {
  final String urlToNewsfeed;
  final String date;
  final String title;
  final String shortDescription;
  final String imageUrl;

  Newsfeed({
    @required this.urlToNewsfeed,
    @required this.date,
    @required this.title,
    @required this.shortDescription,
    @required this.imageUrl,
  });
}
